packages/ods/src/components/modal/tests/rendering/ods-modal.spec.ts

```diff
@@ -0,0 +1,67 @@
+import type { SpecPage } from '@stencil/core/testing';
+import { newSpecPage } from '@stencil/core/testing';
+import { ODS_MODAL_COLOR, OdsModal } from '../../src';
+
+describe('ods-modal rendering', () => {
+  let page: SpecPage;
+  let root: HTMLElement | undefined;
+
+  async function setup(html: string): Promise<void> {
+    page = await newSpecPage({
+      components: [OdsModal],
+      html,
+    });
+
+    root = page.root;
+  }
+
+  describe('attributes', () => {
+    describe('color', () => {
+      it('should be reflected', async() => {
+        const colorValue = ODS_MODAL_COLOR.warning;
+
+        await setup(`<ods-modal color="${colorValue}"></ods-modal>`);
+
+        expect(root?.getAttribute('color')).toBe(colorValue);
+      });
+
+      it('should be set to "information" by default', async() => {
```

Comment:
you can just put `to default value`, otherwise when default value change, it is very likely that the title doesn't get updated alongside the test